server/bot/commands/api.ts

```diff
@@ -224,10 +224,22 @@ export const handleDog: Command = {
       // Fetch a random picture for a sub-breed.
     } else if (args[0] && args[1]) {
       try {
-        const { message } = await (await fetch(
+        let { message } = await (await fetch(
           `http://dog.ceo/api/breed/${args[0].toLowerCase()}/${args[1].toLowerCase()}/images/random`
         )).json()
-        if (!message || message === 'Breed not found') return 'This breed/sub-breed does not exist!'
+        if (!message || message.includes('Breed not found')) {
+          ({ message } = await (await fetch(
+            `http://dog.ceo/api/breed/${args.join('').toLowerCase()}/images/random`
+          )).json())
+          if (!(message.includes('Beed not found'))) {
+            return {
+              embed: { image: { url: message }, color: 0x654321 },
+              content: `🐕 ${args.join('')}`
+            }
+          } else {
+            return 'This breed/sub-breed does not exist!'
+          }
+        }
```

Comment:
There are no dogs with spaces, it should always be joined.
```suggestion
        if (!message || message.includes('Breed not found')) return 'This breed/sub-breed does not exist!'
```